Inputs/GraphQL/gqlQueries.robot

```diff
@@ -67,3 +67,18 @@ getAHUsPropertiesOfSpecificGroup
     [Arguments]    ${group_name}
     ${query}=  set variable  query getGroupsAHUProperties{site{groups: children(selector: {type: Group, name: "General-test"}){name ahus:children(selector: {type: AHU}){ name CoolSource: propString(name: "CoolSource") DesignCapacity: propFloat(name: "DesignCapacity") DesignCop: propFloat(name: "DesignCop")}}}}
     return from keyword    ${query}
+
+#========Queries related to Alarm testcases
+
+getAlarmMsgAHUFailToTurOff
+    ${query}=   set variable   query getAlarmMsgAHUFailTotest { alarms(selector:{type:AhuFailToTurnOff}){ message } }
+    return from keyword    ${query}
+
+getAHUMismatchForAHUFailToTurOffAlarm
+    ${query}=   set variable   query AHUMismatchForAHU { pointCurrent(selector: {name: "SyncFaultStatus"}, filter: {any: [{value: 1}, {value: 2}]}) {SyncFaultStatus: value point { AHU: parent { pathName }}}}
```

Comment:
This also can be renamed in a way to make it generic.
As we are fetching the list of AHUs in Mismatch state, some name like getAHUsInMismatchState or getAHUsWithMismatchState will be good.
This mismatch may happen during other Alarm conditions like 'AHUFailedToTurnON also -(Alarm-02)

Reply:
updated to getAHUsInMismatchState